### DISCUSSION TITLE
Package Roadmap!

### DISCUSSION THREAD

**Discussion by muriloperosa** [muriloperosa/discord-webhook#10] [Q&A] [answered]
Hey folks,

Let's spice up our application roadmap a bit? We want to hear all the awesome suggestions you've got!

If you have ideas for new features, improvements on the current ones, or anything you think will make a difference, feel free to share. The more ideas, the merrier!

Drop your suggestions in the comments below until [deadline date]. Can't wait to see what you've got in mind!

**Comment by jrfernandodasilva** [comment, marked as answer]
Add the Attachments option
as requested in the [issue](https://github.com/muriloperosa/discord-webhook/issues/6)
it would be amazing
